pkg/volume/util/hostutil/hostutil_linux_test.go

```diff
@@ -283,6 +283,9 @@ func TestGetFileType(t *testing.T) {
 
 	for idx, tc := range testCase {
 		path, cleanUpPath, err := tc.setUp()
+		if len(cleanUpPath) > 0 {
```

Comment:
theoretically, there is no need for this check: 

> `func RemoveAll(path string) error`
> RemoveAll removes path and any children it contains. It removes everything it can but returns the first error it encounters. ***If the path does not exist, `RemoveAll` returns `nil` (no error).*** If there is an error, it will be of type *PathError.

Reply:
It sounds good to me.